CHANGELOG.md

```diff
@@ -2,12 +2,57 @@
 
 ## Unreleased
 
+:boom: **Breaking changes**
+
+- Updated back link and breadcrumbs ([PR 1002](https://github.com/nhsuk/nhsuk-frontend/pull/1002))
+
+The breadcrumbs component no longer contains its own `<div class="nhsuk-width-container">` container.
+
+Instead, you should move it inside the existing `<div class="nhsuk-width-container">` container for your overall page, but before the `<main>` tag.
+
+This means that instead this:
+
+```html
+<nav class="nhsuk-breadcrumb" aria-label="Breadcrumb">
+  <div class="nhsuk-width-container">
+    <ol class="nhsuk-breadcrumb__list">
+      <li class="nhsuk-breadcrumb__item"><a class="nhsuk-breadcrumb__link" href="#">Home</a></li>
+    </ol>
+  </div>
+</nav>
+<div class="nhsuk-width-container">
+  <main class="nhsuk-main-wrapper" id="maincontent" role="main">
+    ...
+  </main>
+</div>
+```
+
+You should have this:
+
+```html
+<div class="nhsuk-width-container">
+  <nav class="nhsuk-breadcrumb" aria-label="Breadcrumb">
+    <ol class="nhsuk-breadcrumb__list">
+      <li class="nhsuk-breadcrumb__item"><a class="nhsuk-breadcrumb__link" href="#">Home</a></li>
+    </ol>
+  </nav>
+  <main class="nhsuk-main-wrapper" id="maincontent" role="main">
+    ...
+  </main>
+</div>
+```
+
+The back link should also be placed within the `<div class="nhsuk-width-container">` container but before the `<main>` tag, as before.
```

Comment:
Two things:
* Is this a breaking change? it's unclear if it is or not. Again, might be helpful for more explicit 'you need to do x, y z.
* Include before and after examples?

Reply:
It’s kinda unclear whether this is a breaking change or not, as the existing guidance on the [page template](https://service-manual.nhs.uk/design-system/styles/page-template) suggests having it in this position (within ohs-width-container but before `main`) however the [Back link page](https://service-manual.nhs.uk/design-system/components/back-link) itself is kinda ambiguous, suggesting it could go at the top or the bottom of the page!

Unlike the breadcrumb, the HTML output of the back link component hasn’t changed (as it’s not currently including its own `<div class="nhsuk-width-container">` element), so this feels like less of a breaking change. The only difference is the addition of default margin above.

Reply:
Maybe just make the sentence clearer:

> The back link has not changed position, it should still be placed within the `<div class="nhsuk-width-container">` container but before the `<main>` tag.

Reply:
I think it would be good to include the full markup, or else we could link to the layouts page where people can see them being used?

Reply:
I'm not having second thoughts about "The back link has not changed position" because whilst it's not changing position according to the current page template guidance, it is changing position according to the back link guidance itself (which suggests having it at the bottom).

Also in practice I think a lot of services mistakenly have it within the `<main>` tag so that it's not being skipped by the skip-to-main-content link.

How about something like:

> The back link should also be placed within the `<div class="nhsuk-width-container">` container but before the `<main>` tag.  Previous guidance suggested placing at the bottom of the page, but this has been updated to recommend placing it at the top.

(Then users can read the back link page for the full guidance)

Reply:
The guidance doesn't show in code how to position the back link so I think I would link to layout too.

Reply:
The new guidance in https://github.com/nhsuk/nhsuk-service-manual/pull/2034 says:

> Within the HTML it should appear before the <code>&lt;main&gt;</code> tag. This is so that <a href="/design-system/components/skip-link">skip link</a> skips past the back link to the main content.

Could link to the page template guidance from the What’s new notes too though?

Reply:
Updated the service manual whats new to mention the new position of the breadcrumbs

Reply:
Updated.